README.md

```diff
@@ -171,26 +171,96 @@ If you need to change the Judoscale integration configuration, you can pass a di
 judoscale_rq(redis, extra_config={"LOG_LEVEL": "DEBUG"})
 ```
 
-## Configuring workers
+### Judoscale with RQ and Flask
 
-If you have followed the [RQ on Heroku pattern](https://python-rq.org/patterns/) for setting up your RQ workers on Heroku, you'll have to edit the import statement for `HerokuWorker`:
+The recommended way to initialise Judoscale for RQ is in the application factory:
+
+```py
+judoscale = Judoscale()
+
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object("...") # or however you configure your application
+    app.redis = Redis()
+
+    # Initialise the Judoscale integration for Flask
+    judoscale.init_app(app)
+
+    # Initialise the Judoscale integration for RQ
+    judoscale_rq(app.redis)
+
+    return app
+```
+
+Then, in your worker script, make sure that you create an app, which will initialise the Judoscale integration with RQ. Although not required, it's useful to run the worker within the Flask app context. If you have followed the [RQ on Heroku pattern](https://python-rq.org/patterns/) for setting up your RQ workers on Heroku, your worker script should look something like this:
```

Comment:
> Although not required, it's useful to run the worker within the Flask app context.

Might be obvious to a Flask dev, but can you help me understand what this means and why it's useful?

Reply:
Yeah sure. The app context gives the user access to the `current_app` proxy so you don't have to pass in `app` everywhere and certain properties on the app instance are only available when there is a context to operate in. IIRC database activity, via Flask-SQLAlchemy for instance, also require an app context. Same for rendering views outside of the request-response cycle.

For very simple scenarios the app context is not needed, but as soon as the app gets more complex and you access the DB, render views outside of the request-response cycle etc, you need an app context.